timesketch/frontend-ng/src/components/LeftPanel/SigmaRules.vue

```diff
@@ -61,13 +61,38 @@ export default {
   },
   data: function () {
     return {
-      sigmaRules: [],
-      expanded: false,
+      expanded: true,
       itemsPerPage: 10,
       search: '',
     }
   },
+  methods: {
+    createNewSigmaRule() {
+      // check current router location
+      console.log(this.$route.params.id + 'create Sigma rule')
+      if (this.$route.params.id === 'new') {
+        return
+      }
+      this.$router.push({
+        name: 'Studio',
+        params: {
+          id: 'new',
+          type: 'sigma',
+        },
+      })
+    },
+  },
```

Comment:
Please make this a `<router-link>` element.

Reply:
I could, but then one would push the new method all over again every time the button is hit, which I would prefer to not have from a usability POV